Accessibility.md

```diff
@@ -0,0 +1,32 @@
+# Accessibility
+
+## Motivation
+
+Why we should we focus on a11y?
+What are the benefits?
+Maybe some cons?
+
+## Screen reader
+
+Mac's Voice Over (and alternatives)
+Window's screen readers
+
+## Semantics
+
+What is semantics? (short paragrphs)
+List of semantic elements (or link to extrnal list)
+How to structure a page?
+
+## ARIA
+
+What is ARIA?
+How to use it?
+Links to good examples of using ARIA
+
+## Examples
+
+Do we need this section?
```

Comment:
Here I would expect example implementations of common components we develop in applications (buttons, accordians, carousels, etc.) that showcase how to make them more accessible.